client/scss/layouts/_home.scss

```diff
@@ -7,4 +7,13 @@
     // pull out the icon so it aligns with no-icon text
     margin-inline-start: -1.75em;
   }
+
+  .homepage-panels {
```

Comment:
This styling on this page is pretty minimal and I think we can re-think the fixes and avoid this addition completely.

Why not keep `nice-padding` on the element that is meant to render `nice-padding` and then just remove that class on any inner divs that no longer need it.

I feel like this style addition is fighting too much against existing conventions (yes, I know we may remove nice-padding).

Reply:
Thats a great point for sure, i've actually only added this to ensure any custom added panels using the `construct_homepage_panels` hook that might include nice-padding wouldn't have that extra space. 

I've reverted this now as its probably not needed

Reply:
Just an additional note here. 
https://github.com/wagtail/wagtail/blob/6814d0beea34ca3e868811210ff2210934e72ec3/client/scss/core.scss#L188

We actually say we should not be adding new styles to this file. I had not seen this until now but makes sense so we can move to component styles where possible. 

However, that comment may be out of date haha.